.github/workflows/linters.yml

```diff
@@ -0,0 +1,44 @@
+name: Linters
```

Comment:
- [ ] Please add a proper Readme in your project. README file should be descriptive - all main headers from the [README template](https://github.com/microverseinc/readme-template/blob/master/README.md) should be present in your repo
- it is your choice regarding how much into the details you go in each section of the Readme file should be descriptive
- all main headers from the [README template](https://github.com/microverseinc/readme-template/blob/master/README.md) should be present in a student's repo
- it is your choice regarding how much into the details you want go in each section of the README

Reply:
Noted and fixed. Thanks for the feedback Arslan Bishat